### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by mslinnea**

### CODE REVIEW

File: src/exercise/03.md [hunk 1]
```diff
@@ -2,7 +2,17 @@
...
 
-Elaborate on your learnings here in `src/exercise/03.md`
+Spread Operator
+
+JSX can be written like
+`<div {...props} />` where props is an object containing children and className, for example.
+
+However, for readability, this is better: 
+```
+const props = {children, className};
```
Comment (thischrisblack) [inline src/exercise/03.md]: I'm not sure what this `props` declaration is doing? For the markup below you'd just need `children` and `className` without putting them in an object.

(I know this is just a note but I want to be sure I'm not missing something ...) :)

File: src/exercise/04.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
 ## 📝 Your Notes
 
-Elaborate on your learnings here in `src/exercise/04.md`
+This exercise uses PropTypes. The build doesn't fail if the props aren't correct, but there is a notice in the console.
+
+PropTypes are reactive, while TypeScript is proactive.
```
Comment (thischrisblack) [inline src/exercise/04.md:7]: I like this way of thinking about it!

**Review by thischrisblack — APPROVED** [review]
🐴

File: src/exercise/03.md (hunk 1 above)
Comment (mslinnea) [reply src/exercise/03.md]: oh good point!  no, you aren't missing anything.  I had the markup different before.

**Pull request merged**